### PR TITLE
REBASE: Re-structure user configuration and add more dashboards based on #69

### LINKED ISSUE #67: Adopt to new message definitions
Defect widgets:
- [x] GPS Map
- [x] Spectrum (possible backend fault)

Missing widgets:
- [x] Amplitude (amp, freq1, fre2)
- [x] Velocity (measured, estimated)

### LINKED ISSUE #69: Add dashboards for different program states
Please add the following dashboards to the application:

- [x] Overview (see #68 image)
- [x] Sounding Rocket
  - [x] State Widget
  - [x] Amplitude Graph Widget
  - [x] Spectrum Graph Widget
  - [x] Velocity Graph Widget
- [x] System State
  - [x] State Widget
  - [x] Baro Graph Widget
  - [x] NineDOF Widget
  - [x] Map (when possible)
- [x] Recovery
  - [x] Map Widget (bigg boyyy)

correlates with #67

### PR DESCRIPTION
**Pull request by fussel178**
## Features

- Re-structure user configuration and add more dashboards based on #69
- Implement placeholder widget and add it to Sounding Rocket dashboard
- Implement GPS details widget

closes #69
closes #67

### CODE REVIEW

File: src/widgets/gps-details-widget/widget.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+					alignItems="start"
+				>
+					<Heading marginTop={0} flexGrow={0} level={3}>
+						{title}
+					</Heading>
+					<StatusLight
+						marginTop={-1}
+						variant={isConnected ? 'positive' : 'negative'}
+					>
+						{isConnected ? 'Connected' : 'Disconnected'}
```
Comment (pklaschka) [inline src/widgets/gps-details-widget/widget.tsx:31]: Can we find some more descriptive names here? Maybe something like _Fix established_ and _Low Precision_ or something like that? _Connected_/_Disconnected_ might be confused with the connection status of the client application (or backend application to some other device) 🤔

**Review by pklaschka — CHANGES_REQUESTED** [review]
Thank you very much for your contribution. I will be honest: I don't really like the names in the status light and think they may be a bit confusing, so let's maybe brainstorm about other possibilities there. Other than that, it looks good.

File: src/widgets/gps-details-widget/widget.tsx (hunk 1 above)
Comment (fussel178) [reply src/widgets/gps-details-widget/widget.tsx:31]: How sounds _Located_ and _Interrupted_?

Comment (fussel178) [reply src/widgets/gps-details-widget/widget.tsx:31]: Or _active_ and _suspended_?

Comment (pklaschka) [reply src/widgets/gps-details-widget/widget.tsx:31]: I think that's a good idea. Maybe later, we'll get some even better ideas, but for now, that sounds great 🙂

Comment (fussel178) [reply src/widgets/gps-details-widget/widget.tsx:31]: _Discovered_ and _Lost?

Comment (fussel178) [reply src/widgets/gps-details-widget/widget.tsx]: Applied as _Fix established_ and _Fix pending_ in https://github.com/TelestionTeam/telestion-rocketsound-psc/pull/75/commits/d2f9bbcea5ff7c640fc82d4e0b2798df23b06ee7 :wink:

**Review by pklaschka — APPROVED** [review]
LGTM!

**Pull request merged**